Gir.toml

```diff
@@ -87,6 +87,14 @@ generate = [
     "Gtk.EventSequenceState",
     "Gtk.Expression",
```

Comment:
This is not a `GObject` but a separate fundamental type... so all these have to be manually bound. `glib::ParamSpec` is a good other example for such a thing.

Note that the `glib::Value` related functions for this are `gtk_value_XXX_expression()`.

All the expressions have to be handled specially.

Reply:
See https://github.com/gtk-rs/gtk4/issues/21